src/pages/ImagesModal.js

```diff
@@ -53,33 +58,45 @@ const ImagesModal = (props) => {
       <div className="modal-wrap">
         <div className="container">
           <div className="modal-content-wrap display-flex">
-            <div className="camera-options-wrap display-flex border-radius">
-              <label className="cameraButton">
-                Take photo
-                <input
-                  type="file"
-                  accept="image/*"
-                  capture=""
-                  onChange={imageHandler}
-                  onClick={(e) => (e.target.value = null)}
-                />
-              </label>
-              <label className="cameraButton">
-                Choose from gallery
-                <input
-                  type="file"
-                  accept="image/*"
-                  onChange={imageHandler}
-                  onClick={(e) => (e.target.value = null)}
-                />
+            {isLoading ? (
+              <label className="spinner-wrap display-flex">
+                <span className="lds-spinner">
+                  {[...Array(12).keys()].map(() => (
+                    <span />
+                  ))}
+                </span>
               </label>
-            </div>
-            <div
-              className="cameraButton cancel-button margin-top-bottom border-radius"
-              onClick={onClose}
-            >
-              Cancel
-            </div>
+            ) : (
+              <>
+                <div className="camera-options-wrap display-flex border-radius">
+                  <label className="cameraButton">
+                    Take photo
+                    <input
+                      type="file"
+                      accept="image/*"
+                      capture=""
+                      onChange={imageHandler}
+                      onClick={(e) => (e.target.value = null)}
```

Comment:
strange functionality: I don't think it is a good idea to mutate e.target.value in our function

Reply:
the onClick functionality has been removed